drivers/constants.go

```diff
@@ -9,4 +9,5 @@ const (
 	DriverMySQL    string = "mysql"
 	DriverPostgres string = "postgres"
 	DriverSqlite   string = "sqlite3"
+	DriverMsSQL    string = "sqlserver"
```

Comment:
```suggestion
	DriverMSSQL    string = "sqlserver"
```

While the MySQL acronym uses a lowercase `y`

MSSQL is the expected acronym

https://acronyms.thefreedictionary.com/MSSQL

Reply:
ill make new commit regarding this